app/tests/test_browse.py

```diff
@@ -234,6 +234,152 @@ def test_browse_get_without_filter(
         verify_browse_view_header_row(response.data)
         verify_data_rows(response.data, expected_rows)
 
+    def test_browse_get_with_transferring_body_filter_and_transferring_body_sorting_z_to_a(
```

Comment:
As suggested before, I think we could reduce a lot of this test code by paramterizing the query parameters passed into the client.get request and the expected rows